x-team/auto-ui
# Added not found message inside ApplicantGrid component

**Release Type:** *Non-Breaking Feature* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2476

## Description



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs


- ApplicantGrid component

branch | PR
------ | ------
XP-2476-show-table-in-applicant-not-found| [Auto](https://github.com/x-team/auto/pull/1637)

## Steps to Test or Reproduce
1- Go to the Admin page
2- Choose the tabular view
3- Use the column filter of timeZone and choose a timezone without applicants
4- You should be able to see an empty table with the filter columns and a message below **No results found** 



## Impacted Areas in Application

- Admin page > Tabular and List view [Status not found aplicant]



## Screenshots
![not_found_error](https://user-images.githubusercontent.com/5392202/48443725-77f59b80-e768-11e8-82ed-e33d718507d5.gif)
